pom.xml

```
@@ -31,7 +31,7 @@
     <trimStackTrace>false</trimStackTrace> <!-- surefire -->
     <spotbugs.skip>true</spotbugs.skip>
 
-    <selenium.version>3.141.59</selenium.version>
+    <selenium.version>4.5.0</selenium.version>
```

```suggestion
    <selenium.version>4.6.0</selenium.version>
```

4.5.0 had an important[ thread concurrency issue that caused random flakes](https://github.com/SeleniumHQ/docker-selenium/issues/1507) that we need to pickup.